frontend/src/lib/utils.tsx

```diff
@@ -535,3 +535,23 @@ export function sampleSingle<T>(items: T[]): T[] {
     if (!items.length) throw Error('Items array is empty!')
     return [items[Math.floor(Math.random() * items.length)]]
 }
+
+export function ellipsis(input: string, max: number): string {
+    return input.length > max ? input.substr(0, max) + '...' : input
```

Comment:
I think we actually don't even need this. The "description" field in the `plugin.json` field should be just a line of text, not more. In case it gets too long, we should add "..." via CSS and not by splitting the string. Removed.